stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/notifications/medianotifications/MediaNotificationHandler.kt

```diff
@@ -19,13 +19,20 @@ package io.getstream.video.android.core.notifications.medianotifications
 import androidx.core.app.NotificationCompat
 import io.getstream.video.android.model.StreamCallId
 
+@Deprecated(message = "This interface is deprecated. Use the notification interceptors instead.")
```

Comment:
I feel maybe we should mention these interface names `StreamNotificationBuilderInterceptors` and `StreamNotificationUpdateInterceptors` in our deprecated message. Wdyt?